src/main/java/step2/domain/LottoTier.java

```diff
@@ -0,0 +1,31 @@
+package step2.domain;
+
+public enum LottoTier {
+
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```
언더스코어를 통해 가독성을 높일 수 있답니다.